Assets/MRTK/Core/Inspectors/Utilities/InspectorUIUtility.cs

```diff
@@ -604,5 +604,58 @@ public static int GetOptionsIndex(SerializedProperty options, string selection)
 
             return 0;
         }
+
+        /// <summary>
+        /// Draws the contents of a scriptable inline inside a foldout. Depending on if there's an actual scriptable
+        /// linked, the values will be greyed out or editable in case the scriptable is created inside the serialized object.
+        /// </summary>
+        static public bool DrawScriptableFoldout<T>(SerializedProperty scriptable, string description, bool isExpanded) where T : ScriptableObject
+        {
```

Comment:
This is extremely useful, tested out `DrawScriptableFoldout` for the interactable changes and it works great.